baselines/ippo/config.py

```diff
@@ -5,9 +5,10 @@
 @dataclass
 class ExperimentConfig:
     """Configurations for experiments."""
+
     # DATASET & DEVICE
     data_dir: str = "maps_10"
-    train_on_k_unique_scenes: int = 500  # If set to a non-zero value K, randomly samples K unique scenarios for a total of num_worlds scenes
+    train_on_k_unique_scenes = 10  # If set to a non-zero value K, randomly samples K unique scenarios for a total of num_worlds scenes
```

Comment:
Did you accidentally remove the type? 
```suggestion
    train_on_k_unique_scenes : int = 10  # If set to a non-zero value K, randomly samples K unique scenarios for a total of num_worlds scenes
```